Task Liverpool/src/main/java/pages/SearchResultsPage.java

```diff
@@ -0,0 +1,92 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchResultsPage extends BasePage{
+    private By numberOfResultsFromUpperRightLabel = By.cssSelector(".a-plp-results-title span");
+    private By searchResultsDisplayed = By.cssSelector("li.m-product__card");
+    private By numberOfPagesBy = By.xpath("//a[contains(@class, 'page-link') and string-length(text()) > 0]");
+    private By listViewButton = By.cssSelector(".icon-vistalista");
+    private By sortByDropDownList = By.cssSelector(".col-lg-9 #sortby");
+    private By sortByPrice = By.cssSelector(".col-lg-9 .dropdown-menu [datahref=\"/tienda/sortPrice|1\"]");
+    private By productPrice = By.cssSelector(".m-product__card .a-card-discount");
+    private By rightArrow = By.cssSelector(".page-link .icon-arrow_right");
+
+    JavascriptExecutor js = (JavascriptExecutor) getWebDriver();
+
+    public SearchResultsPage(WebDriver webDriver) {
+        super(webDriver);
+    }
+
+    public int getNumberOfResultsFromUpperRightLabel(){
+        return Integer.parseInt(getElementText(numberOfResultsFromUpperRightLabel));
+    }
+
+    public int getTotalNumberOfResultsDisplayed() {
+        int totalResults = findElements(searchResultsDisplayed).size();
+        List<WebElement> pages = findElements(numberOfPagesBy);
+        int lastPage = Integer.parseInt(pages.get(pages.size()-1).getText());
+
+        //Logic for clicking on the Next Results Page Numbers with a For
+        for (int i = 1; i < lastPage; i++) {
+            String productId = getElementAttribute(findElement(searchResultsDisplayed), "data-prodid");
+            By firstResult = By.cssSelector("li.m-product__card[data-prodid=\"" + productId + "\"]");
+            js.executeScript("window.scrollTo(0, document.body.scrollHeight)");
```

Comment:
I think an scroll is a comon action, maybe move this to basePage as a method, and something like scrillToElement and receive the element that you want to scroll to

Reply:
fixed.